scripts/update_project_mailing_lists.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
For posterity, a docstring describing the purpose of the script would be welcome.